cartographer_ros/cartographer_ros/msg_conversion.cc

```diff
@@ -235,4 +235,12 @@ geometry_msgs::Pose ToGeometryMsgPose(const Rigid3d& rigid3d) {
   return pose;
 }
 
+geometry_msgs::Point ToGeometryMsgPoint(const Rigid3d& rigid3d) {
```

Comment:
As I said, this should take `Eigen::Vector3d` since it only uses that part. Pass in `rigid3d.translation()` at the call sites.